lib/UnoCore/Targets/Android/app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,38 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="@(Activity.Package)">
 
+    #if @(Project.Android.SupportsScreens.Resizable:IsSet) || @(Project.Android.SupportsScreens.SmallScreens:IsSet) || @(Project.Android.SupportsScreens.NormalScreens:IsSet) || @(Project.Android.SupportsScreens.LargeScreens:IsSet) || @(Project.Android.SupportsScreens.XlargeScreens:IsSet) || @(Project.Android.SupportsScreens.AnyDensity:IsSet) || @(Project.Android.SupportsScreens.RequiresSmallestWidthDp:IsSet) || @(Project.Android.SupportsScreens.CompatibleWidthLimitDp:IsSet) || @(Project.Android.SupportsScreens.LargestWidthLimitDp:IsSet)
+    <supports-screens 
+                  #if @(Project.Android.SupportsScreens.Resizable:IsSet)
+                  android:resizeable="@(Project.Android.SupportsScreens.Resizable:ToLower)"
+                  #endif
+                  #if @(Project.Android.SupportsScreens.SmallScreens:IsSet)
+                  android:smallScreens="@(Project.Android.SupportsScreens.SmallScreens:ToLower)"
+                  #endif
+                  #if @(Project.Android.SupportsScreens.NormalScreens:IsSet)
+                  android:normalScreens="@(Project.Android.SupportsScreens.NormalScreens:ToLower)"
+                  #endif
+                  #if @(Project.Android.SupportsScreens.LargeScreens:IsSet)
+                  android:largeScreens="@(Project.Android.SupportsScreens.LargeScreens:ToLower)"
+                  #endif
+                  #if @(Project.Android.SupportsScreens.XlargeScreens:IsSet)
```

Comment:
Maybe `XlargeScreens` would look better if it was `XLargeScreens`? That feels a little bit more natural to me at least.

Though, project property names are case-insensitive so either of those will compile...

Reply:
Thanks for spotting it, done